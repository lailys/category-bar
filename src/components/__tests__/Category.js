import React from "react"
import renderer from "react-test-renderer"
import Category from "../category"



it("Category should renders correctly", () => {
  const tree = renderer.create( < Category / > ).toJSON()
  expect(tree).toMatchSnapshot()
})
