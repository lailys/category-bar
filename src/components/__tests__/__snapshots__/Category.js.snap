// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Category renders correctly 1`] = `
<div
  className="sc-bdnylx cQpAGy"
>
  <a
    className="sc-gtssRu jgDMBM"
    href="/category/"
  />
</div>
`;

exports[`Category should renders correctly 1`] = `
<div
  className="sc-bdnylx cQpAGy"
>
  <a
    className="sc-gtssRu jgDMBM"
    href="/category/"
  />
</div>
`;
